etc/spec/ARCHITECTURE.md

```diff
@@ -119,12 +109,44 @@ The server can publish the following messages:
   4. `datetime_start`: A unix epoch in ms which indicates the datetime the
      message started being typed.
 
+The real-time server exposes an HTTP endpoint at `https://ting.gr/api/v1` with the following
+URLs:
+
+* A PATCH operation on `/message`: Sends an update on the message with an id equal to <messageid>.
+  The message can be sent from the user to a channel or to another user directly.
```

Comment:
Please assume the reader of ARCH has read the SPEC. This sentence here is needless and confusing – are you talking about a ting message, a piece of text exchanged between users and channels? Then this should be on the SPEC. Are you talking about a protocol message, as should be on the ARCH? Then how can a channel receive such messages?

I suggest you remove this sentence completely.